lib/SelectableItem/SelectableItem.less

```diff
@@ -3,20 +3,15 @@
 	padding-left: (@moon-item-indent + @moon-spotlight-outset);
 
 	&:before {
-		content: '';
+		content: '\2022';
```

Comment:
I see a few issues with using a bullet character in this instance, with regards to its size. First, the bullet character here is larger than the original bullet character, ignoring @moon-selectable-item-indicator-width and @moon-selectable-item-indicator-height. Second, the size of the bullet is inconsistent with the size of Radio Item indicator. Maybe a UX person can provide some input?